src/include/migraphx/op/reduce_mean.hpp

```diff
@@ -1,110 +1,36 @@
 #ifndef MIGRAPHX_GUARD_OPERATORS_MEAN_HPP
 #define MIGRAPHX_GUARD_OPERATORS_MEAN_HPP
 
-#include <migraphx/check_shapes.hpp>
-#include <migraphx/argument.hpp>
-#include <migraphx/shape_for_each.hpp>
-#include <migraphx/par_for.hpp>
-#include <migraphx/config.hpp>
-#include <vector>
+#include <migraphx/op/reduce_op.hpp>
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace op {
 
-struct reduce_mean
+struct mean_op
 {
-    std::vector<std::int64_t> axes{};
-
-    template <class Self, class F>
-    static auto reflect(Self& self, F f)
+    auto op() const
```

Comment:
These methods should be in the `reduce_mean` class not in a seperate class.

Reply:
fixed